cmd/helm/testdata/testcharts/alpine/templates/alpine-pod.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
     # The "release" convention makes it easy to tie a release to all of the
     # Kubernetes resources that were created as part of that release.
     app.kubernetes.io/instance: {{.Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
```

Comment:
The reason this wasn't included for these test data before was that the Charts themselves don't define an `AppVersion` so this is going to set it to an empty string. It's ok not to include something if there will be no value. This thought applies to all the label changes on this particular testdata chart.

Reply:
I've added `appVersion` to the `Chart.yaml`